app/src/include/firebase/app.h

```diff
@@ -697,7 +697,9 @@ class App {
   /// @param library Name of the library to register as a user of the Firebase
   /// C++ SDK.
   /// @param version Version of the library being registered.
-  static void RegisterLibrary(const char* library, const char* version);
+  /// @param context Platform specific resource. Ex. for Android, this is JNIEnv
```

Comment:
@param platform_resource to match the variable name

Reply:
Done